src/domain/event/EventPage.tsx

```diff
@@ -36,17 +36,20 @@ const EventPage = ({ event, children, success }: EventProps) => {
       >
         {success}
         <div className={styles.backButtonWrapper}>
-          <Button
-            aria-label={t('common.backButton.label')}
-            className={styles.backButton}
-            onClick={() => history.goBack()}
-          >
-            <Icon
-              src={backIcon}
-              className={styles.backButtonIcon}
-              alt={t('common.backButton.label')}
-            />
-          </Button>
+          <div className={styles.backButtonInnerWrapper}>
+            <Button
+              variant="secondary"
+              aria-label={t('common.backButton.label')}
+              className={styles.backButton}
+              onClick={() => history.goBack()}
+            >
+              <Icon
+                src={backIcon}
```

Comment:
Did you consider using icons from HDS? https://hds.hel.fi/storybook/react/?path=/story/components-icons-ui--iconarrowleft

Reply:
I changed a couple of icons and then decided to wait with the rest until this PR is merged, it has grown so large. So a follow up PR could be for changing the icons.